python/hsfs/feature_view.py

```diff
@@ -625,11 +580,11 @@ def label(self):
 
         Can be a composite of multiple features.
```

Comment:
getter/setter name "label" vs variable name "labels"

Reply:
Is line 505 corect? "fv.label = [feature.name for feature in features if feature.label]"